PluginController/EntityPluginController.php

```diff
@@ -397,4 +389,10 @@ protected function doGetPaymentInstruction($id)
 
         return $paymentInstruction;
     }
+
+    protected function doRollback()
+    {
+        $this->entityManager->getConnection()->rollback();
+        $this->entityManager->close();
```

Comment:
To clarify: I know that the behavior is not introduced by this PR and existed for a long time. But still it is a good time to change it.